clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -590,6 +591,58 @@ static Error getSYCLDeviceLibs(SmallVector<std::string, 16> &DeviceLibFiles,
   return Error::success();
 }
 
+static Error getDeviceLibsForLTO(SmallVector<OffloadFile> &DeviceLibs,
```

Comment:
Can we just call this getDeviceLibs or some generic name and use it in https://github.com/intel/llvm/blob/70ee0e04c960c06fc69323f7597232700cb34957/clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp#L1364 as well?
Is that possible?
Thanks

Reply:
Marking this as resolved...as this is a RFC PR and I do not think we need to address code improvements here.